conda.recipe/meta.yaml

```diff
@@ -32,20 +32,22 @@ requirements:
     - pip
     - ruamel_yaml >=0.11.14,<0.17
     - setuptools >=31.0.1
+    - toolz >=0.8.1
   run:
     - python
     - conda-package-handling >=1.3.0
-    - cytoolz >=0.8.1
     - menuinst >=1.4.11,<2         # [win]
     - pycosat >=0.6.3
     - pyopenssl >=16.2.0
     - requests >=2.18.4,<3
     - ruamel_yaml >=0.11.14,<0.17
     - setuptools >=31.0.1
+    - toolz >=0.8.1
```

Comment:
```suggestion
    # conda-build to drop cytoolz dependency (https://github.com/conda/conda-build/pull/4556)
    # once that occurs this can be relaxed to toolz (https://github.com/conda/conda/pull/11700)
    - cytoolz >=0.8.1
```

Reply:
No longer necessary with https://github.com/conda/conda/pull/11704